src/spikeinterface/core/node_pipeline.py

```diff
@@ -467,31 +474,91 @@ def run_node_pipeline(
     nodes,
     job_kwargs,
     job_name="pipeline",
-    mp_context=None,
+    #mp_context=None,
```

Comment:
Shouldn't we leave this for people that use positional?

Reply:
I think it was a mistake because it should be in job_kwargs no ?
Alessio is it you that add this mp_context directly here ?

Reply:
I think it was you @samuelgarcia